external-builds/README.md

```diff
@@ -0,0 +1,56 @@
+# rockbuilder
+
+Rockbuilder provides a configuration file based way of building
+one ore multiple projects. Project can be application, library or
+some other buildable or installable source code.
+
+Configuration files are stored in the projects-directory and there exist
+2 types of configuration files.
+
+## project list configuration files
+
+project list configuration files can be used to store a list of projects
+that are wanted to build at the same time. At the moment this is hardcoded
+to be a projects/core_apps.cfg but it is planned that in future there
+can be multiple different project lists.
```

Comment:
Should document the config file format here, at least linking to https://docs.python.org/3/library/configparser.html which is used to parse them.